tasks/install-ubuntu-cuda-repo.yml

```diff
@@ -30,13 +33,6 @@
     name: nouveau
     state: present
 
-- name: add repo
-  apt_repository:
-    repo: "deb {{ nvidia_driver_ubuntu_cuda_repo_baseurl }} /"
-    update_cache: yes
-  environment: "{{proxy_env if proxy_env is defined else {}}}"
-  when: nvidia_driver_add_repos | bool
-
```

Comment:
are we removing this because we always use the nvidia/cuda packages from ubuntu repos?

Reply:
We remove this because the `cuda-keyring` package is actually installing the repo files itself! And having both of these causes failures.